src/audio-view.js

```diff
@@ -0,0 +1,193 @@
+//      disk: {
+//      },
+      simon: {
+        panels: [0,1,2].map(stripId => _.range(10).map(panelId => `sounds/Game_03/G03_LED_${("0"+(stripId*10+panelId+1)).slice(-2)}.wav`)),
+        success: 'sounds/Game_03/G03_Success_01.wav',
+        failure: 'sounds/Game_03/G03_Negative_01.wav',
+        show: 'sounds/Game_03/G03_Light_Show_01.wav'
+      }
+    };
+
+    // Traverse this.sounds and replace the filenames with valid sound objects.
+    this._promises = [];
+    this._traverse(this.sounds);
+
+    // _loadSound() will create promises. We call the callback once all promises resolve
+    console.log(`${this._promises.length} promises`);
+    Promise.all(this._promises)
+      // Don't listen to events until we've loaded all sounds
+      .then(() => this.store.on(SculptureStore.EVENT_CHANGE, this._handleChanges.bind(this)))
+      .then(() => callback(null))
+      .catch(callback.bind(null));
+  }
+
+  // Traverses sound config objects and replaces nodes with valid, loaded, sounds
+  _traverse(node) {
+    for (let key in node) {
+      const value = node[key];
+      let sound;
+      if (typeof value === 'string') sound = node[key] = new Sound({url: value});
+      else if (value instanceof Sound) sound = value;
+      if (sound) this._loadSound(sound);
+      else this._traverse(value);
+    }
+  }
+
+  // Returns a promise to provide a comple, usable sound.
+  _loadSound(sound) {
+    const promise = sound.load();
+    this._promises.push(promise);
+    return promise;
+  }
+
+  _handleChanges(changes) {
+    if (this._animating) {
+      return;
+    }
+
+    if (changes.currentGame === GAMES.HANDSHAKE) this.sounds.alone.ambient.play();
+
+    this._handleHandshakeChanges(changes);
+    this._handleStatusChanges(changes);
+    this._handleLightChanges(changes);
+  }
+
+  _handleHandshakeChanges(changes) {
+    if (changes.handshakes) {
+      // Did someone shake my hand?
+      if (changes.handshakes[this.config.username]) {
+        this.sounds.alone.ambient.stop();
+        this.sounds.alone.handshake.play();
+      }
+    }
+  }
+
+  _handleStatusChanges(changes) {
+    if (changes.status) {
+      let statusSounds;
+
+      if (this.store.isPlayingMoleGame) {
+        statusSounds = {
+          [SculptureStore.STATUS_SUCCESS]: this.sounds.mole.success,
+          [SculptureStore.STATUS_FAILURE]: this.sounds.mole.failure
+        };
+      }
+      if (this.store.isPlayingSimonGame) {
+        statusSounds = {
+          [SculptureStore.STATUS_SUCCESS]: this.sounds.simon.success,
+          [SculptureStore.STATUS_FAILURE]: this.sounds.simon.failure
+        };
+      }
+
+      const statusSound = statusSounds[changes.status];
+      if (statusSound) statusSound.play();
+    }
+  }
+
+  _handleLightChanges(changes) {
+    const lightChanges = changes.lights;
+    if (!lightChanges || !this.store.isReady) {
+      return;
+    }
+    
+    if (this.store.isPlayingMoleGame) {
+      const lightArray = this.lightArray;
```

Comment:
By event handlers do you mean change handlers? (Sorry I'm not sure what code you're referring to.) 
If you're referring to what I think you are referring to:
Even if it is temporary code, moving the conditions inside the loops will dramatically decrease the amount of code you have in that handler and make everyone's lives easier for when this temporary code becomes "real" code.